data/beatscript_scene_data.c

```diff
@@ -2,17 +2,17 @@
 #include "beatscript_scene_data.h"
 
 
-/* BeatScript Scene Data */
+/* Beatscript Scene Data */
```

Comment:
Any reason why we're going with "Beatscript" instead of "BeatScript"?

Reply:
I mean it's not called TickFlow, I'm not sure why it was ever capitalised like that to begin with

Reply:
fair enough